docs/Developers/Review.rst

```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is ready
+         to be merged,or
```

Comment:
`merged,or` is incorrect grammar

Reply:
Please check if it's correct now @jayvdb